app/uk/gov/hmrc/plasticpackagingtax/registration/forms/contact/AddressMapper.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms.contact
+
+import play.api.data.Forms.{mapping, optional}
+import play.api.data.{Form, Forms}
+
+trait AddressMapper {
+
+  val formToModel
```

Comment:
`formToModel` is not needed - you can just use `Address.apply` as the data structure of the mapping and the default constructor are the same.

Ditto fot `modelToForm`

Reply:
👍